database/migrations/2024_03_11_000000_add_origin_and_destination_id_to_trips.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+/**
+ * "SQLite doesn't support multiple calls to dropColumn / renameColumn in a single modification."
+ * -> so small steps here. (╯°□°)╯︵ ┻━┻
+ */
+return new class extends Migration
+{
+    public function up(): void {
```

Comment:
This migration should be split into multiple migrations and maybe moved into transactions.